### ISSUE TITLE
Accessibility: Video embeds cannot be tabbed to

### ISSUE THREAD

**Issue by LWinterberg** [audacity/audacity.github.io#13]

**Comment by DilsonsPickles** [comment]
Fixed here: https://github.com/audacity/audacity.github.io/actions/runs/6527788674
